scripts/setup-adapters.sh

```diff
@@ -31,6 +31,53 @@ function install_aws-sdk-cpp {
   cmake_install -DCMAKE_BUILD_TYPE=Debug -DBUILD_SHARED_LIBS:BOOL=OFF -DMINIMIZE_SIZE:BOOL=ON -DENABLE_TESTING:BOOL=OFF -DBUILD_ONLY:STRING="s3;identity-management" -DCMAKE_INSTALL_PREFIX="${DEPENDENCY_DIR}/install"
 }
 
+function install_gcs-sdk-cpp {
+  # Install gcs dependencies
+  # https://github.com/googleapis/google-cloud-cpp/blob/main/doc/packaging.md#required-libraries
+
+  # abseil-cpp
+  github_checkout abseil/abseil-cpp 20220623.1 --depth 1
+  sed -i 's/^#define ABSL_OPTION_USE_\(.*\) 2/#define ABSL_OPTION_USE_\1 0/' "absl/base/options.h"
+  cmake_install -DBUILD_SHARED_LIBS=OFF \
+    -DABSL_BUILD_TESTING=OFF
+
+  # protobuf
+  github_checkout protocolbuffers/protobuf v21.12 --depth 1
```

Comment:
Protobuf is already required and part of resolve_dependencies, is there a reason why 21.12 is required over 21.4 ?

Reply:
It was one of the dependencies that GCS stated back then when I originally added that function. Let me check, probably I can get rid of it.